src/components/HomePage/Modals/Congratulation.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import { useDispatch, useSelector } from "react-redux";
+import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+import { AppText, Heading } from '../../shared';
+import './Modal.scss';
+import { toggleCurrentGenresCreator } from "../../../redux/actions/actionCreators/actionCreators";
+import { congratsModalStatusSelector } from "../../../redux/selector/selectors";
+
+export const Congratulation: FC = () => {
+    const dispatch = useDispatch();
+    const open = useSelector(congratsModalStatusSelector);
+    const toggleModal = () => {
+        dispatch(toggleCurrentGenresCreator());
+    };
+
+    return (
+        <Modal
+            backdrop={false}
+            centered
+            fullscreen="md"
+            scrollable
+            size="md"
+            className="special-modal"
+            isOpen={open}
+        >
+            <ModalHeader toggle={toggleModal}>
+                <div className="d-flex flex-row align-items-center">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="34" height="34" fill="#f65261"
```

Comment:
Let's consider importing images instead of embedding

Reply:
Fixed, thanks